css/main.css

```diff
@@ -488,3 +488,128 @@ footer {
     padding: 0px 250px 80px 210px;
   }
 }
+
+/***********************/
+/* EMAIL CAPTURE FORM */
+/***********************/
+
+.section-cta {
+  padding: 4.8rem 15.4rem;
+}
+
+.container {
```

Comment:
Remove the `.container` class, since there are no style properties assigned to it.

Reply:
Hey, @priyanshk20 and @DeRaowl I have removed the .container class in both index.html and main.css. Also deleted the excess comments present on the files.